charts/customer-center/README.md

```diff
@@ -0,0 +1,158 @@
+| frontend.readinessProbe.successThreshold | int | `1` | Minimum consecutive successes for the probe to be considered successful after having failed |
+| frontend.readinessProbe.timeoutSeconds | int | `5` | Number of seconds after which the probe times out |
+| frontend.replicaCount | int | `1` | (int) Number of Backend replicas |
+| frontend.resources | object | `{}` | Resource limits for frontend |
+| frontend.service.externalPort | int | `80` | External Port of frontend service |
+| frontend.service.internalPort | int | `80` | Internal Port of frontend service |
+| frontend.service.name | string | `"customer-center-frontend"` | Frontend service name |
+| frontend.service.type | string | `"ClusterIP"` | Frontend service type |
+| ingress.annotations | object | `{}` | Ingress annotations |
+| ingress.certificate.certFilename | string | `"tls.crt"` | Cert file name |
+| ingress.certificate.certKeyFilename | string | `"tls.key"` | Cert key filename |
+| ingress.certificate.dnsNames[0] | string | `"my.example.com"` |  |
+| ingress.certificate.issuerRef.group | string | `"cert-manager.io"` |  |
+| ingress.certificate.issuerRef.kind | string | `"ClusterIssuer"` |  |
+| ingress.certificate.issuerRef.name | string | `"letsencrypt-prod"` |  |
+| ingress.certificate.name | string | `"customer-center"` | Name of certificate |
+| ingress.certificate.secret | string | `"customer-center-tls"` | Name of TLS secret |
+| ingress.enabled | bool | `true` | Enable ingress for customer-center |
+| ingress.hosts | list | `["my.example.com"]` | Ingress hostnames |
+| ingress.tls | list | `[{"hosts":["my.example.com"],"secretName":"customer-center-tls"}]` | Ingress TLS options |
+| postgresql.enabled | bool | `true` | Enable PostgreSQL for persistence |
+| postgresql.image | string | `nil` |  |
+| postgresql.ingress.enabled | bool | `false` | Enable ingress |
+| postgresql.ingress.loadBalancerSourceRanges | list | `[]` | Whitelist specific IP ranges |
+| postgresql.ingress.selector | object | `{"app.kubernetes.io/name":"postgresql","role":"slave"}` | Service selector labels |
+| postgresql.postgresqlDatabase | string | `"db"` |  |
+| postgresql.postgresqlPassword | string | `"password"` | Set a password for PostgreSQL |
+| postgresql.postgresqlUsername | string | `"user"` |  |
+| postgresql.tls.certFilename | string | `"tls.crt"` | Cert file name |
+| postgresql.tls.certKeyFilename | string | `"tls.key"` | Cert key filename |
+| postgresql.tls.certificate.dnsNames | list | `[]` | DNS names of certificate |
+| postgresql.tls.certificate.issuerRef | object | cert-manager | Issuer ref |
+| postgresql.tls.certificate.name | string | `"psql-server"` | Name of certificate |
+| postgresql.tls.certificatesSecret | string | `"psql-server"` | Name of TLS secret |
+| postgresql.tls.enabled | bool | `false` | Enable TLS for Postgresql |
+| redis.cluster.enabled | bool | `false` | Use master-slave topology |
```

Comment:
We should not shadow values from dependency charts unless we document why the shadow is needed.

Reply:
@fkm-adfinis any input on this?